### PR TITLE
Update index.mdx

### PR DESCRIPTION
**Pull request by vickimcfadden**
Updating resources page per new content. Still awaiting some feedback so not ready for review.

### CODE REVIEW

File: src/content/resources/index.mdx [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+description: "A full list of resources designed to help buyers and sellers navigate the OASIS+ contract: explore software applications, tools, and documentation."
 order: 0
 ---
 import TableBorderless from "@components/TableBorderless.astro"
 import TableHeaderBorderless from "@components/TableHeaderBorderless.astro"
 import TableCellBorderless from "@components/TableCellBorderless.astro"
 export const components = {table: TableBorderless, th: TableHeaderBorderless, td: TableCellBorderless}
 
-We have resources to help you navigate the OASIS+ contract; as part of this, we have included descriptions of all software applications, tools, and documentation related to the contract. If you do not see what you need, please reach out to us at [oasisplus@gsa.gov](mailto:oasisplus@gsa.gov).
+We have resources to help buyers and sellers
```
Comment (klohman) [inline src/content/resources/index.mdx:11]: Looks like there is some extra space here that's not needed?

**Comment by rosamundtgov** [comment]
Updated resources page with updated body copy and metadata. Linked to the OASIS+ fact sheet. 

Update existing resources page (see [#63](https://github.com/orgs/GSA/projects/111/views/1?filterQuery=resources&pane=issue&itemId=52452608)) with [new content](https://docs.google.com/document/d/1uaFZULg_IqJplogKvAXYwSpD9ST4hrBSG-21B8qnV_c/edit#heading=h.iygv7gu3pcbe).

Specifically: Reference buyers' guide (printable version) and a a couple links from Develop and Issue the Solicitation (see copy doc)

[Find samples and templates](https://buy.gsa.gov/find-samples-templates-tips?sort_by=title&sort_order=asc&page=1)
[eBuy Open](https://www.ebuy.gsa.gov/ebuyopen/)
Buyers’ guide (printable)[/ordering-guide]
Metadata should read: A full list of resources designed to help buyers and sellers navigate the OASIS+ contract: explore software applications, tools, and documentation.

Body text should read: We have resources to help buyers and sellers navigate the OASIS+ contract; as part of this, we have included descriptions of all software applications, tools, and documentation related to the contract. If you do not see what you need, please reach out to us at [oasisplus@gsa.gov](mailto:oasisplus@gsa.gov).

Linked fact sheet on [OASIS+ Fact Sheet (PDF]

File: src/content/resources/index.mdx [hunk 2]
```diff
@@ -19,20 +20,39 @@ We have resources to help you navigate the OASIS+ contract; as part of this, we
...
+| [Symphony - The task order tool that works for you](https://www.youtube.com/watch?v=cQ4zVfA8MAY) | Video tutorial that gives an overview of Symphony.|
+| [Symphony Task Order System - How to register](https://gettingstarted.apexlogic.com/support/solutions/articles/35000177795-registering-as-a-government-user)| Provides instructions for how to register with Symphony. |
+| Getting started - Frequently Asked Questions | Frequently asked questions about Symphony. |
+| [How to Use Symphony Task Order Management for OASIS+ (video)](https://www.youtube.com/watch?v=LXkqNV88FJI)| Learn how to submit proposals and capture information in Symphony, the database of record for all stakeholders. 
+
+Note: this video references OASIS, but the information remains relevant to OASIS+.|
+| [Symphony Task Order Management for OASIS+ Customers (PDF)](https://www.gsa.gov/system/files/Final_-_Symphony_Customer_Rollout_Presentation_%282022-03-17%29_.pdf) | Learn more about Symphony’s benefits, role in the ordering process, and additional resources related to training and support.
+
```
Comment (klohman) [inline src/content/resources/index.mdx:43]: In the preview, this section isn't showing as part of the table (and I think it's intended to?) 
Screenshot 2024-05-31 at 2 05 33 PM

**Review by klohman — COMMENTED** [review]
Couple comments

**Review by rosamundtgov — COMMENTED** [review]
Updated based on klohman's responses and correcting the OASIS+ fact sheet file name -

**Review by rosamundtgov — COMMENTED** [review]
Updated to fix my bad coding -

**Comment by rosamundtgov** [comment]
changes made, ready for review

File: src/content/resources/index.mdx (hunk 1 above)
Comment (rosamundtgov) [reply src/content/resources/index.mdx]: resolved

File: src/content/resources/index.mdx (hunk 2 above)
Comment (rosamundtgov) [reply src/content/resources/index.mdx]: resolved

**Review by rosamundtgov — COMMENTED** [review]
Updated to correct formatting

**Review by rosamundtgov — COMMENTED** [review]
Corrected formatting due to build errors

**Review by klohman — APPROVED** [review]
👍

**Pull request merged**